compiler/compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -152,6 +152,10 @@
     private ArrayList<String> weakFrameworks;
     @ElementList(required = false, entry = "path")
     private ArrayList<File> frameworkPaths;
+    @ElementList(required = false, entry = "extension")
+    private ArrayList<String> appExtensions;
+    @ElementList(required = false, entry = "path")
```

Comment:
Isn't there already an entry "path"?
Or is this not a problem?

Reply:
this annotation defines that field it attached to going to be list where all elements are enclosed in <path> tag.